src/utils/types.js

```diff
@@ -147,6 +147,7 @@ const types = {
 		'nauticalmiles',
 	]),
 	aggregationData: array,
+	showClearAll: oneOf(['never', 'always', 'default', true, false]),
```

Comment:
@ShahAnuj2610 Can you define these values as constants and use these constants in `ReactiveSearch` or `ReactiveCore` to make the comparison?